internal/propelauth/client.go

```diff
@@ -19,6 +21,23 @@ type PropelAuthClient struct {
 	ApiKey     string
 }
 
+type PropelAuthApiError struct {
+	ErrorCode string `json:"error_code"`
+	UserFacingError string `json:"user_facing_error"`
```

Comment:
may want to also include `field_errors` and `user_facing_errors` in case it's a validation error we'r getting back.

Reply:
done :)